src/pages/Home/Home.tsx

```diff
@@ -19,24 +20,26 @@ export const HomePage = (): ReactElement => {
       <Helmet>
         <title>Home - Answer King</title>
       </Helmet>
-      <div className="grid grid-cols-2 gap-2">
-        <div className="ml-44 mt-20 flex flex-col items-center justify-center">
-          <div className="mb-10 text-center text-[80px] text-white">
-            <h3 className="font-poly italic leading-tight">
-              The
-              <img src={Logo} alt="logo" className="ml-5 mb-4 inline-block h-12" />
-              <span className="font-poppins not-italic">nswer</span> to <br></br> your cravings
-            </h3>
+      <PageTransition>
```

Comment:
Do you think we could avoid the code repetition by making a general Page component, with a prop for the title and a prop for the contents that are children of the PageTransition component on each page?

Then the home page would become something like <Page title="Home - Answer King" content=<HomeContent /> />